imobil.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Imobil extends Model
+{      
+    use HasFactory;
+    protected $fillable = [
+        'owner_id',
+        'type',
+        'price',
+        'rooms_nr',
+        'building_type',
+        'square_mp',
+        'floor',
+        'bath',
+        'balcony',
+        'description',
+        'created_at',
+    ];
+
+
+    public function address()
+    {
+        return $this->belongsTo(Address::class);
+    }
+    public function owner()
+    {
+        return $this->belongsTo(Owner::class);
+    }
+
+    public function benefit()
+    {
+        return $this->hasMany(Benefit::class);
```

Comment:
this is a relation many to many
https://laravel.com/docs/8.x/eloquent-relationships#many-to-many-model-structure